.github/workflows/check.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: Build Schema
         run: yarn schema
 
+      - name: Check Dockerfile
+        run: docker build .
```

Comment:
is this enough as check ?
I just run the build, so it would fail on syntax errrors in the Dockerfile.
To do more, would be difficult

Reply:
Looks good. Maybe let's run it in a separate task and we could run a simple build inside of it to see that it succeeds.

Reply:
What do you mean by "simple build" ?    "yarn build " ?

Reply:
"Docker build" vs "vega lite build" ?

Reply:
I meant `yarn build` and making sure it was successful.

Reply:
done